src/cli/subcommands/snapshot_cmd.rs

```diff
@@ -55,13 +55,16 @@ pub enum SnapshotCommands {
 
     /// Validates the snapshot.
     Validate {
-        /// Number of block headers to validate from the tip
-        #[arg(long, default_value = "2000")]
-        recent_stateroots: i64,
-        /// Validate already computed tipsets at given EPOCH,
-        /// use a negative value -N to validate the last N EPOCH(s) starting at HEAD.
+        /// Number of recent epochs to scan for broken links
+        #[arg(long, default_value_t = 2000)]
+        check_links: i64,
+        /// Assert the snapshot belongs to this network. If left blank, the
+        /// network will be inferred before executing messages.
         #[arg(long)]
-        validate_tipsets: Option<i64>,
+        check_network: Option<crate::networks::NetworkChain>,
+        /// Number of recent epochs to scan for bad messages/transactions
+        #[arg(long, default_value_t = 60)]
+        check_stateroots: i64,
```

Comment:
Do we expect negative values?

Reply:
We expect the unexpected!

Replaced with `u32`.